Include/internal/pycore_fileutils.h

```diff
@@ -318,6 +318,8 @@ PyAPI_FUNC(int) _PyLong_FileDescriptor_Converter(PyObject *, void *);
 // Export for test_peg_generator
 PyAPI_FUNC(char*) _Py_UniversalNewlineFgetsWithSize(char *, int, FILE*, PyObject *, size_t*);
 
+PyAPI_FUNC(int) _PyFile_Flush(PyObject *);
```

Comment:
Please don't export a function unless it's technically needed:

```suggestion
extern int _PyFile_Flush(PyObject *);
```

I prefer to make new private functions as private as possible :-)